Helpers/RLMRealm+Notifications.m

```diff
@@ -65,4 +65,26 @@ - (void)deleteObjectsWithNotification:(id<NSFastEnumeration>)array
     [self deleteObjects:array];
 }
 
+#pragma mark - private helpers
+
+- (BOOL)rbq_containsObject:(RLMObject *)object {
+    if (!object) {
+        return NO;
+    }
+    
+    Class objectClass = [object class];
```

Comment:
Yeah I guess it does and already you need the Utilities repo since that is where RBQSafeRealmObject lives, so I am ok with putting it there.